src/staff/Staff.tsx

```diff
@@ -1,31 +1,68 @@
 import React from "react";
 
-type StaffProps = {
-  y?: number;
-  width?: number;
-};
+import "./Staff.css";
 
-export default function Staff({ y = 0, width = 2000 }: StaffProps) {
+/**
+ * We're extending Staff with SVGAttributes to keep DRY
+ */
+export interface StaffProps extends React.SVGAttributes<HTMLOrSVGElement> {
+}
+
+export default function Staff( {
+  x = 0,
+  y = 0,
+  viewBox = `0 0 100 100`,
+  height = "100",
+  width = "100",
+  preserveAspectRatio = "none"
+}: StaffProps ) {
   return (
     <svg
       data-testid="staff"
-      x="0"
+      x={x}
       y={y}
-      viewBox={`0 0 ${width} 101`}
+      viewBox={viewBox}
       width={width}
-      height={101}
-      style={{ overflow: "visible" }}
+      height={height}
+      preserveAspectRatio={preserveAspectRatio}
     >
-      <line x1="0" y1="0" x2={width} strokeWidth="2" y2="0" stroke="black" />
-      <line x1="0" y1="25" x2={width} strokeWidth="2" y2="25" stroke="black" />
-      <line x1="0" y1="50" x2={width} strokeWidth="2" y2="50" stroke="black" />
-      <line x1="0" y1="75" x2={width} strokeWidth="2" y2="75" stroke="black" />
       <line
         x1="0"
-        y1="100"
-        x2={width}
-        strokeWidth="2"
-        y2="100"
+        y1="1"
+        x2="100"
+        y2="1"
+        strokeWidth="1"
```

Comment:
Hey! Just tested manually and is perfectly backwards compatible 🥇   🎉  

The only thing I would still keep as before is the `strokeWidth` with value 2. I'm concerned that it may look too thin when zoomed out, and I can foresee a few situations where it would be zoomed out to small sizes. You can see in the screenshot the lines look like greyed out, even at a relatively large size.

If we can fix this and the other details this would be ready for merge!

![Screenshot 2020-12-16 at 16 39 32](https://user-images.githubusercontent.com/583564/102378166-68875300-3fbd-11eb-83b7-5b6dad206f11.png)

Reply:
Sure, Amy! I will update PR with default strokewidth as 2 and other suggestions.

Thank you!

Reply:
Configured line strokewitdh with default value as 2 in e204311972269c7afcc78c494796eaae78fc659c!